flight/targets/cc3d/fw/pios_board.c

```diff
@@ -107,14 +107,8 @@ void PIOS_Board_Init(void) {
 	/* Delay system */
 	PIOS_DELAY_Init();
 
-	const struct pios_board_info * bdinfo = &pios_board_info_blob;
```

Comment:
Are you sure we should remove this?  It seems nice to refuse to start on original CopterControl.